datacollection/openapi/X3/README.md

```diff
@@ -5,3 +5,10 @@
 NASA's clarifications to the ASTM interface spec are documented in ![this base doc copied from ASTM](https://github.com/nasa/uam-apis/blob/master/datacollection/nasa-astm-utm.yaml)
 
 You can compare the ASTM interface spec with NASA's clarifications using ![this compare URL](https://github.com/nasa/uam-apis/compare/implementation_2020q2_752454c..master)
+
+Data Pipeline is hosted by AWS API Gateway which has restrictions ![as documented here](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-known-issues.html).  In particular date-time is
```

Comment:
As a disinterested engineer, I'm not interested *in general* with the restrictions on the OpenAPI spec imposes, nor am I interested in *why* three decimal places are the limit for my timespecs (although it is super-important to us).
ECMA 262? Important. The regular expression that I have to conform to? Important.

Reply:
agreed. Removed.